docs/changelog/msi_removal.yaml

```diff
@@ -0,0 +1,14 @@
+pr: -1
```

Comment:
I'm on the fence about omitting `pr` here. It's not actually a required field, as we allow pr-less files for security notices or known issues, but we later validate the area. I'm inclined to live with this on `7.17` and address it properly in `v8.x` when it comes up again.